### ISSUE TITLE
Downloading Satellite data

### ISSUE THREAD

**Issue by jptokaya** [UBA-DE-Emissionsituation/space-emissions#13]
Some package I came across that can be handy for us to download satellite data: https://sentinelsat.readthedocs.io/en/stable/#
This works for Sentinel satellites in general and hence also TROPOMI.

**Comment by kevin-hausmann** [comment]
This looks really nice! And it fits very well with the technologies we have already selected (pandas and geopandas) - great find! It also seems to be under really active development while being stable enough to be useful.
Please note that on the CODE-DE machines some satellite data will be already accessible on the file system and we should not waste all the time needed for download. The calculation method modules we build will have to be context aware and configurable in that way.

**Comment by jptokaya** [comment]
Yeah, I was playing around with it and it seems to work decently. Agreed that we don't spend to much time on the tool for downloading data then! The check should be made if the data is already on the CODE-DE machine or a download needs to be performed. We can think about a smart way to handle either of these situations
